less/_components/bootstrap/buttons.less

```diff
@@ -15,7 +15,12 @@
 
 /* Custom buttons */
 .btn {
-  text-transform: @btn-text-transform;
+  & {
+    text-transform: @btn-text-transform;
+    font-weight: 600;
+    font-size: 18px;
```

Comment:
Let's create a custom variable for this value

Reply:
We still need a new variable for the font-size for btn.

Also, I would of expected you to change our default value of @btn-font-weight;. I believe this is the intent of the bug?

Reply:
We already had it defined at 700 (which is equal to bold font), but never actually set into our button classes.